Source/Scene/BrdfLutGenerator.js

```diff
@@ -39,35 +36,28 @@ function createCommand(generator, context) {
   generator._drawCommand = drawCommand;
 }
 
-function createFramebuffer(generator, context) {
-  var colorTexture = new Texture({
-    context: context,
-    width: 256,
-    height: 256,
-    pixelFormat: PixelFormat.RGBA,
-    pixelDatatype: PixelDatatype.UNSIGNED_BYTE,
-    sampler: Sampler.NEAREST,
-  });
-
-  generator._colorTexture = colorTexture;
-
-  var framebuffer = new Framebuffer({
-    context: context,
-    colorTextures: [colorTexture],
-    destroyAttachments: false,
-  });
-
-  generator._framebuffer = framebuffer;
-}
-
 BrdfLutGenerator.prototype.update = function (frameState) {
-  if (!defined(this._colorTexture)) {
+  if (!defined(this.colorTexture)) {
```

Comment:
Nitpick - since the getter is now simply returning `_colorTexture` there's no need to use the getter for internal code, the private member variable can be accessed directly.

```suggestion
  if (!defined(this._colorTexture)) {
```